docs/versioned/spark/spark-procedures.md

```diff
@@ -280,6 +280,11 @@ See the [`RewriteDataFiles` Javadoc](../../../javadoc/{{% icebergVersion %}}/org
 and <br/> [`SortStrategy` Javadoc](../../../javadoc/{{% icebergVersion %}}/org/apache/iceberg/actions/SortStrategy.html#field.summary)
 for list of all the supported options for this action.
 
+!!! Note
+    This procedure can also read the delete files created by merge-on-read mode if present
+    and rewrite to new data files by masking the result from delete files.
+    By default, delete files will not be considered for compaction. Need to configure `delete-file-threshold` in options.
```

Comment:
For the last sentence, I think writing it out may read better, what do you think of this?:
```diff
- Need to configure `delete-file-threshold` in options.
+ In order to change this behavior, you need to set `delete-file-threshold`.
```

Reply:
I think we need to avoid, `you`, `we` in docs.

I adopted your sentence with small modification (removing 'you')

In order to change this behavior, need to set `delete-file-threshold`.
Also added same sentence in maintainance.md file.